mos-platform/common/ldscripts/noinit-sections.ld

```diff
@@ -1,3 +1,3 @@
 *(.noinit .noinit.* NULL INIT ZPSAVE)
 /* This helps ensure all chunk sizes have the low bit free for use as a flag. */
-__heap_start = ALIGN(2);
+__heap_start = ALIGN(., 2);
```

Comment:
Great find; definitely add a comment here though that CP/M-65 depends on this, and why. Otherwise, anyone seeing this would have to find this PR via git blame.

Reply:
Done. This is, AFAICT, the only place in the common code where ALIGN is used like this. I don't know if any of the other platforms have relocatable binaries (has someone approached you about O65 binaries?) but I believe this is the only instance affecting me.